src/style.css

```diff
@@ -0,0 +1,77 @@
+.boxed {
+  
+  border: 5px solid #9fccfc ;
+  border-radius: 50px;
+ background-color: #eaf4ff;
+
+  
+ 
+}
+.topnav{
+    position: fixed;
```

Comment:
Typically, you don't want to use `position: fixed` for layouts, it makes the styles more brittle and less responsive (i.e. it won't look as nice if someone resizes the browser). One really helpful tool for complex layouts like this is [flex](https://css-tricks.com/snippets/css/a-guide-to-flexbox/). Flex is a super helpful set of properties all centered around distributing space in a container. There's this super fun game called [flexbox froggy](https://flexboxfroggy.com/) that makes this super easy to use and work with.